src/main/java/com/internet/shop/web/filters/AuthenticationFilter.java

```diff
@@ -30,13 +30,14 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         HttpServletResponse resp = (HttpServletResponse) response;
 
         String url = req.getServletPath();
-        if (url.equals("/users/login") || url.equals("/users/registration")) {
+        if (url.equals("/users/login") || url.equals("/users/registration")
```

Comment:
let's also refactor AuthenticationFilter as proposed in the checklist

Reply:
Done